puppet/zulip/files/logrotate/zulip

```diff
@@ -1,8 +1,13 @@
-/var/log/zulip/server.log /var/log/zulip/workers.log /var/log/zulip/manage.log {
+/var/log/zulip/*.log {
```

Comment:
So this is the big change; in particular, I imagine the important thing you're doing here is fixing the fact that supervisor's own logs (which are rotated automatically) don't get compressed; is that correct?  

I'm not sure what the right answer is for how to make that work, but I think we don't want to have logrotate conflicting with supervisor here as to who is rotating the supervisor log files.  I think quite possibly the right answer is what I did in 6355f26b91a983610129926acef432e0688eb44d, which is to dramatically decrease the total size of supervisor logs (since they tend to be lower-value than the main server.log / errors.log files)